### DISCUSSION TITLE
Metric and Relative Monocular Depth Estimation: An Overview. Fine-Tuning Depth Anything V2 👐 📚

### DISCUSSION THREAD

**Discussion by isayoften** [huggingface/computer-vision-course#321] [08. 3D Vision, Scene Rendering and Reconstruction (NERF, GQN, etc.)]
Hello!
Can we take the content of my blog-post for this unit? I think the post is very educational and fits well into this topic. I also didn't find any good guides on the topic of depth estimation.
https://huggingface.co/blog/Isayoften/monocular-depth-estimation-guide

**Comment by johko** [comment]
Hey @isayoften, sorry for the late reply. The content looks awesome! Feel free to create a PR with an mdx file containing your write-up

**Reply by isayoften** [reply]
That's great! I was about to ask, is it normal that on HF the formulas display correctly but on GitHub they don't? I've tried changing the syntax on Github, but with no luck. 
![image](https://github.com/user-attachments/assets/51227e2f-a796-450e-90c2-fdb44ee89f76)
![image](https://github.com/user-attachments/assets/a95c6ce4-6f30-4823-9957-a83f2f0e259c)

**Reply by johko** [reply]
Sorry I didn't reply earlier @isayoften and actually I was stumbling upon that formula problem as well now, as I was reviewing your PR.
The main problem right now is that due to a broken deployment pipeline I also cannot see the PR pre-view on HF side, from which I could tell if everything is rendered correctly. So for now I will ignore the formulas in the review and hope to have the pipeline and preview fixed very soon
